BenchmarkSamples/benchmark/build.gradle

```diff
@@ -11,7 +11,16 @@ android {
         versionName "1.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
 
+    testBuildType "benchmark"
```

Comment:
If you're importing a project fresh, does studio understand to update default build type?

I assume not - let's update the debug buildType instead - that makes it easier to just import and run.

May still be useful to show how to do this in an :app project...

What would you think about (in a follow up) adding an :app module to this project, which *just* shows how to get benchmarking running in an app module, with one trivial benchmark. It would show you the 'testBuildType' stuff, as well as explain the structure in the build.gradle, and explain how you need to select the buildType inside of studio.

Reply:
I can add the trivial android app benchmark sample in a separate project :smiley: